test/load_test/locustfile.py

```diff
@@ -1,139 +0,0 @@
-# Copyright (c) Microsoft Corporation.
```

Comment:
See https://github.com/microsoft/scitt-ccf-ledger/issues/37. Can we keep those files and disable mypy validation via some annotation until we fix them?